custom-namespaces.yaml

```
@@ -23,6 +23,10 @@ combined_browser_metrics:
       type: table_view
       tables:
         - table: moz-fx-data-shared-prod.telemetry.active_users_aggregates
+    active_users_aggregates_device:
+      type: table_view
+      tables:
+        - table: moz-fx-data-shared-prod.telemetryactive_users_aggregates_device
```

(that was actually the wrong suggestion haha)
```suggestion
        - table: mozdata.telemetry.active_users_aggregates_device
```